.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
 script:
     - npm test
     - env
+    - eslint
```

Comment:
For reference: https://github.com/mozilla-magnet/magnet-metadata-service/blob/master/package.json#L7